companies/sportsdirect.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
SportsDirect also has stores in cities which may collect data. But I guess these will have to be requested at the respective local administrations. I will look into that.

Reply:
Yes, SportsDirect Retail Ltd. is just held by the SportsDirect international plc. https://en.wikipedia.org/wiki/Sports_Direct